frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarOverviewTab.tsx

```diff
@@ -17,6 +18,7 @@ export function PlayerSidebarPersonTab(): JSX.Element {
             <div className="font-bold bg-bg-light px-2 border-b py-3">
```

Comment:
One concern I have here is with scrolling. Ideally I'd like it to be able to scroll together - table and overview as well, and not have the grid and person display fixed on top.

I can adjust the styles of the table (either based on a prop, or just target with a selector) to enable the whole tab to scroll, or is there an example somewhere where this is nicely done maybe?

https://github.com/user-attachments/assets/3148b2a4-c652-4a30-b98e-b6a401028de3

Reply:
You could move the metadata above the person and then the separate scrolling wouldn't look too strange.

The only place I can immediately think of funky scrolling fun is in the toolbar but that will be hacky rather than a good example 🤣

Reply:
<img width="462" alt="Screenshot 2024-09-25 at 10 46 44" src="https://github.com/user-attachments/assets/6fdcf8ec-2baa-468d-aa5a-f2deb0f20df4">

the keypresses number being right up against errors makes it harder to scan this
you could left align the whole thing 

so instead it says

| 169 clicks      | 211 keypresses    | 27 errors  |